docs/src/main/paradox/docs/fusion/architecture.md

```diff
@@ -28,7 +28,17 @@ Expect the SubApp feature and its functionality will expand and change in the ne
 
 ## Plugins
 
-Plugins are ways to render resources. You can find more about them @ref:[here](plugins.md). It is important to note 
-that the plugin repository is hosted separately from Nexus Fusion. Nexus Fusion will request a Plugin Manifest from 
-this repository at run time, and fetch plugins to render during run time based on a config. Both the plugins, the 
+Plugins are ways to render resources. You can find more about them @ref:[here](plugins.md). It is important to note
+that the plugin repository is hosted separately from Nexus Fusion. Nexus Fusion will request a Plugin Manifest from
+this repository at run time, and fetch plugins to render during run time based on a config. Both the plugins, the
 configuration, and the manifest should be hosted somewhere Nexus Fusion can request it.
+
+## Customization
+
+You can customize the Header of Nexus Fusion by setting up the following environment variables:
+
+- `LOGO_IMG`: Url for an image to be used as application logo in the Header, for example, `https://drive.boogle.com/jnsjdnsjs`
+- `LOGO_LINK`: Url for the logo, for example, `https://www.epfl.ch`
+- If you use Nexus Forge, it is possible to include a `Forge templates` button by providing the url as `FORGE_LINK`, for example, `https://some-url.hi`
+
+The full list of environment variables can be found @link:[here](https://github.com/BlueBrain/nexus-web/blob/main/README.md#env-variables-list).
```

Comment:
I think the link should be any of the following:
- https://github.com/BlueBrain/nexus-web#env-variables-list
- https://github.com/BlueBrain/nexus-web/blob/main/README.md#env-variables-list

Reply:
Looks like we are using the second option here already?

Reply:
That doesn't seem to work for some weird reason. Please remove the ```@link``` just for that one